sql/003-create-artist.sql

```diff
@@ -0,0 +1,14 @@
+use `music-db`;
+
+DROP TABLE IF EXISTS `artist`;
+
+CREATE TABLE `artist`
+(
+    `id`       int(11) NOT NULL AUTO_INCREMENT,
+    `first_name`     varchar(255),
+    `second_name`    varchar(255),
+    `discs` varchar(255),
```

Comment:
`discs` column should not exist, because there should already be a foreign key on discs table referencing this artist.